pom.xml

```diff
@@ -86,11 +86,6 @@
             <name>osiam-releases</name>
             <url>https://api.bintray.com/maven/osiam/OSIAM/org.osiam:osiam/;publish=1</url>
         </repository>
-        <snapshotRepository>
-            <id>snapshots</id>
-            <name>osiam-snapshots</name>
-            <url>https://oss.jfrog.org/artifactory/oss-snapshot-local</url>
-        </snapshotRepository>
```

Comment:
Removing this leads to uploading the snapshot to Bintray releases repo:

```
Uploading: https://api.bintray.com/maven/osiam/OSIAM/org.osiam:osiam/;publish=1/org/osiam/osiam/3.0-SNAPSHOT/osiam-3.0-20160608.081458-1.war
```

I don't think that this is the intended behavior.